doc/data/messages/t/too-many-lines/details.rst

```diff
@@ -1 +1 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
+Used when a module has too many lines, reducing its readability.
```

Comment:
You really need to **actually** check the result before pushing. I told you at least twice already, the description from the checker is automatically added to the doc. What does the checker contain ? "Used when a module has too many lines, reducing its readability." Now what is going to be in the result ? You should generate locally but here goes:

```
Description:

Used when a module has too many lines, reducing its readability.

....

Additional details:

Used when a module has too many lines, reducing its readability.

Created by the [format](https://github.com/PyCQA/pylint/blob/main/pylint/checkers/format.py) checker.
```
So what's wrong here ? Yes, the text is duplicated. Do I have to point this out each time ?

Also we have 40 concurrent runners for the whole organization, wasting runner time by pushing new commits all the time before event checking the result locally is impacting **dozen of persons**, please be more considerate